contrib/helloimage/podman_hello_world.bash

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+###
+#  ASCII art by the incomparable Máirín Duffy,
+#  duffy@redhat.com, Twitter: @mairin
+#  January 2022
+###
+echo "                             "
+echo "! ... Hello Podman World ...!"
+echo "                             "
+echo "         .--\"--.            "
+echo "       / -     - \\          "
+echo "      / (O)   (O) \\         "
+echo "   ~~~| -=(,Y,)=- |          "
+echo "    .---. /\`  \\   |~~      "
+echo " ~/  o  o \\~~~~.----. ~~    "
+echo "  | =(X)= |~  / (O (O) \\    "
+echo "   ~~~~~~~  ~| =(Y_)=-  |    "
+echo "  ~~~~    ~~~|   U      |~~  "
```

Comment:
Can we add some links here, maybe docs.podman.io and github

Reply:
Good thought.  added GitHub, docs, podman.io, and Twitter links.